src/mattack_actors.cpp

```diff
@@ -1273,7 +1273,6 @@ void gun_actor::shoot( monster &z, const tripoint &target, const gun_mode_id &mo
     }
 
     tmp.set_wielded_item( gun );
-    tmp.i_add( item( "UPS_off", calendar::turn, 1000 ) );
```

Comment:
Haha wtf, when a monster shoots we gin up a fake npc to actually do the shooting?